esx_service/utils/auth_data_test.py

```diff
@@ -0,0 +1,286 @@
+        tenant1.set_default_datastore_and_privileges(self.auth_mgr.conn, default_datastore, default_privileges)                              
+
+    
+    def test_add_datastore_access_privileges(self):
+        vms = [(str(uuid.uuid4()), 'vm1')]
+        privileges = []
+        
+        default_datastore, default_privileges = self.get_default_datastore_and_privileges()
+        tenant1 = self.auth_mgr.create_tenant('tenant1', 'Some tenant', default_datastore,
+                                              default_privileges, vms, privileges)
+        self.assertTrue(uuid.UUID(tenant1.id))
+        
+        vms = [(str(uuid.uuid4()), 'vm2')]
+        privileges = []
+        tenant2 = self.auth_mgr.create_tenant('tenant2', 'Some tenant', default_datastore,
+                                              default_privileges, vms, privileges)
+        self.assertTrue(uuid.UUID(tenant2.id))
+        
+        privileges = [{'datastore': 'datastore1',
+                       'global_visibility': 0,
+                       'create_volume': 0,
+                       'delete_volume': 0,
+                       'mount_volume': 0,
+                       'max_volume_size': 0,
+                       'usage_quota': 0}]
+
+        tenant1.set_datastore_access_privileges(self.auth_mgr.conn, privileges)
+
+        privileges = [{'datastore': 'datastore1',
+                        'global_visibility': 0,
+                        'create_volume': 1,
+                        'delete_volume': 1,
+                        'mount_volume': 0,
+                        'max_volume_size': 0,
+                        'usage_quota': 0}]
+        
+        tenant1.set_datastore_access_privileges(self.auth_mgr.conn, privileges)
+    
+
+    def test_list_tenants(self):
+        vms = [(str(uuid.uuid4()), 'vm1')]
+        privileges = []
+        default_datastore, default_privileges = self.get_default_datastore_and_privileges()
+        tenant1 = self.auth_mgr.create_tenant('tenant1', 'Some tenant', default_datastore,
+                                              default_privileges, vms, privileges)
+        self.assertTrue(uuid.UUID(tenant1.id))
+        
+        vms = [(str(uuid.uuid4()), 'vm2'), (str(uuid.uuid4()), 'vm3')]
+        privileges = []
+        tenant2 = self.auth_mgr.create_tenant('tenant2', 'Some tenant', default_datastore,
+                                              default_privileges, vms, privileges)
+        self.assertTrue(uuid.UUID(tenant2.id))
+        
+        privileges = [{'datastore': 'datastore1',
+                       'global_visibility': 0,
+                       'create_volume': 0,
+                       'delete_volume': 0,
+                       'mount_volume': 0,
+                       'max_volume_size': 0,
+                       'usage_quota': 0}]
+
+        tenant1.set_datastore_access_privileges(self.auth_mgr.conn, privileges)
+        tenants_list = self.auth_mgr.list_tenants()
+        for tenant in tenants_list:
```

Comment:
is this print needed for test automation?  I'd prefer to have expected_results and compare it with actual_results